src/services/recim/activity.ts

```diff
@@ -32,19 +34,26 @@ type CreatedActivity = BaseActivity & {
 };
 
 type UploadActivity = BaseActivity & {
+  SportID: number;
+  TypeID: number;
   Logo: string;
 };
 
 type PatchActivity = BaseActivity & {
+  SportID: number;
+  TypeID: number;
   StatusID: number;
   Logo: string;
   Completed: boolean;
 };
 
 //Activity Routes
-const createActivity = (newActivity: UploadActivity): Promise<CreatedActivity> =>
+const createActivity = async (newActivity: UploadActivity): Promise<CreatedActivity> =>
```

Comment:
You don't need the `async` here, since you aren't using `await` at all. Since `http.get` already returns a Promise, it is already asynchronous by default. In fact, adding `async` here is purely a performance reduction because it adds another, unnecessary Promise wrapper around the `http.get` Promse.